crates/test_utils/src/fixture.rs

```diff
@@ -95,16 +95,7 @@ impl Fixture {
             }
         }
 
-        Fixture {
-            path,
-            text: String::new(),
-            krate: krate,
```

Comment:
I changed this to `krate,`, which caused it to be converted to a single line. Not a redundant clone, but figured it'd still be welcome (: